addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -10,6 +10,8 @@
                 'o-temporary': attachment and attachment.isTemporary,
                 'o-viewable': attachment and attachment.isViewable,
             }" t-att-title="attachment ? attachment.displayName : undefined" t-att-data-attachment-local-id="attachment ? attachment.localId : undefined"
+                t-att-aria-label="attachment ? attachment.displayName : undefined"
+                role="group"
```

Comment:
what is the goal of group in summary?

Reply:
It helps to label sections, in this case one per attachment. When navigating using the keyboard, you jump to attachment buttons (e.g. download) but, without this, you don't know what attachment the current button refers to.

With this, buttons are verbalized as follows (one per tab):
- Attachment 1: Download
- Remove
- Attachment 2: Download
- Remove
- ... And so on

I changed it by menu and menuitems for a similar result but a more accurate semantic